### PR TITLE
Temporary stub

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "bem-core": "2.6.0",
     "bem-components": "2.1.0",
-    "bem-history": "git://github.com/bem/bem-history.git#v2"
+    "bem-history": "git://github.com/bem/bem-history.git#107bc24bcfe8995ea42608de21561ef0457ca797"
```
Comment (tadatuta) [inline bower.json:9]: any reason not to use 3.0.0?

Comment (gela-d) [inline bower.json:9]: @tadatuta Yes, details on issue BEMINFO-1111

Comment (tadatuta) [inline bower.json:9]: @gela-d are you sure that there's the same error on **v3.0.0** (not `v2`)?

Comment (gela-d) [inline bower.json:9]: @tadatuta yes, we added that time v3.0.0. Empirically we found that our problem in https://github.com/bem/bem-history/commit/1a1e8bb18eeefe655f4b13978125c7a79c280c48 and this commit is in v3.0.0

**Pull request merged**